recipes/gltf-sdk/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class GltfSdkConan(ConanFile):
+    name = "gltf-sdk"
```

Comment:
Given that gltf is an open standard and a sdk is a common term, this name is pretty generic. Do we want to prefix it?

And since we shortened `microsoft` previously to `ms` we should keep that https://conan.io/center/ms-gsl

So, should we name it `ms-gltf-sdk`?

```suggestion
    name = "ms-gltf-sdk"
```

Reply:
Actually it's what I did in my testing repo https://github.com/SpaceIm/conan-ms-gltf-sdk, but I was not sure.

Reply:
done